docs/templates/navigation.html

```diff
@@ -0,0 +1,124 @@
+                                    style="width: 1000px; height: 500px; border:0; transform: scale(1); overflow:hidden;"
+                                    sandbox="allow-scripts allow-same-origin">
+                                </iframe>
+                            </div>
+                        </div>
+                        <div class="navigation-container flex-col-wrap-center">
+                            <h2>Tabs</h2>
+                            <div class="flex-col-wrap-center">
+                                <div class="tabs-container">
+                                    <div class="tab">TAB ONE</div>
+                                    <div class="tab">TAB TWO</div>
+                                    <div class="tab active">ACTIVE TAB</div>
+                                </div>
+                            </div>
+                            <div class="flex-col-wrap-center">
+                                <div class="tabs-container">
+                                    <div class="tab"><i class="fas fa-clock"></i><span>TAB ONE</span></div>
+                                    <div class="tab"><i class="fas fa-paper-plane"></i><span>TAB TWO</span></div>
+                                    <div class="tab active"><i class="fas fa-heart"></i><span>ACTIVE TAB</span></div>
+                                </div>
+                            </div>
+                            <div class="code-snippet">
+                                <iframe
+                                src="https://carbon.now.sh/embed?bg=rgba%28249%2C250%2C252%2C1%29&t=one-light&wt=none&l=auto&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522tabs-container%2522%253E%250A%2520%2520%2520%2520%253Cdiv%2520class%253D%2522tab%2522%253ETAB%2520ONE%253C%252Fdiv%253E%250A%2520%2520%2520%2520%253Cdiv%2520class%253D%2522tab%2522%253ETAB%2520TWO%253C%252Fdiv%253E%250A%2520%2520%2520%2520%253Cdiv%2520class%253D%2522tab%2520active%2522%253EACTIVE%2520TAB%253C%252Fdiv%253E%250A%253C%252Fdiv%253E"
+                                style="width: 600px; height: 400px; border:0; transform: scale(1); overflow:hidden;"
+                                sandbox="allow-scripts allow-same-origin">
```

Comment:
It would be really great if you can avoid inline styling here.

Reply:
Thanks for the review. :)